### PR TITLE
DOC use pepy-tech badge in README instead of conda-forge one

### PR DESCRIPTION
**Pull request by MarcoGorelli**
cf Gitter conversation

- [ ] closes #xxxx
- [ ] tests added / passed
- [ ] Ensure all linting tests pass, see [here](https://pandas.pydata.org/pandas-docs/dev/development/contributing.html#code-standards) for how to run them
- [ ] whatsnew entry

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,7 +12,7 @@
 [![License](https://img.shields.io/pypi/l/pandas.svg)](https://github.com/pandas-dev/pandas/blob/master/LICENSE)
 [![Azure Build Status](https://dev.azure.com/pandas-dev/pandas/_apis/build/status/pandas-dev.pandas?branch=master)](https://dev.azure.com/pandas-dev/pandas/_build/latest?definitionId=1&branch=master)
 [![Coverage](https://codecov.io/github/pandas-dev/pandas/coverage.svg?branch=master)](https://codecov.io/gh/pandas-dev/pandas)
-[![Downloads](https://anaconda.org/conda-forge/pandas/badges/downloads.svg)](https://pandas.pydata.org)
+[![Downloads](https://pepy.tech/badge/pandas/month)](https://pepy.tech/project/pandas)
```
Comment (bashtage) [inline README.md:15]: These are different things.  anaconda.org downloads are distinct from PyPI, which pepy is counting. Perhaps they should be there any labeled PyPI download and the other Anaconda.org Downloads.

Comment (MarcoGorelli) [reply README.md:15]: yeah the reason for suggesting this is that, if pandas is going to show downloads, it's probably better to show the PyPI ones, as the conda-forge ones are only a small fraction (about 2%)

not sure if there's a way to change the label, will check

Comment (fangchenli) [reply README.md:15]: We could put both badges here. Maybe change the name to `Conda Downloads` and `PyPI Downloads`.

Comment (jreback) [reply README.md:15]: yeah can we add both badges? (and change names)

Comment (MarcoGorelli) [reply README.md]: I've edited this badge to read "PyPI downloads per month"

I don't see how to edit the conda-forge one (https://anaconda.org/conda-forge/pandas/badges), and IMO showing "PyPI downloads per month: 62M" and "Total downloads: 18M" next to each other wouldn't inspire much confidence in a data analysis library 😄 

So, my suggestion is to not how the conda downloads until we can customize its text (though if someone finds to do that already, please do let me know!)

**Comment by jreback** [comment]
@MarcoGorelli status of this?

**Comment by MarcoGorelli** [comment]
> @MarcoGorelli status of this?

I've edited this badge to read "PyPI downloads per month"

I don't see how to edit the conda-forge one (https://anaconda.org/conda-forge/pandas/badges)

So, my suggestion is to just remove the conda downloads count, unless someone can suggest a way to customize its badge's text

**Comment by jreback** [comment]
sure

**Pull request merged**